eno-core/src/main/java/fr/insee/eno/core/utils/PoguesUtils.java

```diff
@@ -32,4 +34,28 @@ public static Stream<QuestionType> poguesQuestionStream(SequenceType poguesSeque
                 .filter(QuestionType.class::isInstance).map(QuestionType.class::cast);
     }
 
+    private static List<ResponseType> getPoguesTableResponse(QuestionType tableQuestion, boolean noDataCells){
+        List<String> coordinateOfNoDataCells = tableQuestion.getResponseStructure()
+                .getAttribute().stream()
+                .filter(a ->"NoDataByDefinition".equals(a.getAttributeValue()))
```

Comment:
On a un magic word qui traîne ici : à refacto dans une constante du style

```java
private static final String POGUES_NO_DATA_ATTRIBUTE = "NoDataByDefinition"
``` 

au niveau de la classe

Reply:
oui c'est vrai qu'in fine la bonne chose à faire c'est de corriger dans Pogues-Model ; en attendant on peut mettre ce palliatif côté Eno qui coûte pas grand chose